check_stability.py

```diff
@@ -353,6 +353,57 @@ def get_files_changed():
             for item in files[:-1].split("\0")]
 
 
+def get_affected_testfiles(files_changed):
+    affected_testfiles = []
+    # FIXME: treat ".xml" files as test files? or support files? or both? or neither?
+    testfile_extensions = [".html", ".htm", ".xhtml", ".xht", ".xml", ".svg"]
```

Comment:
> Couldn't we use the manifest to work out what's a test file and what's a support file?

How? Just by checking if manifest has a `path` value (or `url` value?) that matches the pathname of the changed file? (and then just stopping if it does, because that means it’s a test file, not a support file)